test/utils/wasm3_engine.cpp

```diff
@@ -0,0 +1,106 @@
+#include "source/wasm3.h"
```

Comment:
Have you tried to use include_dir with "source" and then include `<wasm3.h>`?

Reply:
I had `set(include_dir ${source_dir}/source)` but that caused a cmake configuration error claiming no such directory. Initially it worked but not after a while.

Reply:
```
-- Configuring done
CMake Error in test/utils/CMakeLists.txt:
  Imported target "wasm3::wasm3" includes non-existent path

    "/Users/alex/Projects/fizzy/b/_deps/src/wasm3/source"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.
```

Reply:
I will try it later.